### ISSUE TITLE
Rename the "Curve centerline extraction" module

### ISSUE THREAD

**Issue by lassoan** [vmtk/SlicerExtension-VMTK#63]
It seems that we need to rename the "Curve centerline extraction" module to something more specific, as the current name is too similar to the base "Extract centerline" module and this confuses users - see https://discourse.slicer.org/t/new-module-extract-centerline-in-slicervmtk-extension/12117/23?u=lassoan

**Comment by chir-set** [comment]
OK, the next step is to find another name : Curve centerline helper? 'Fiducial centerline extraction' should also be renamed on the same pattern.

**Comment by lassoan** [comment]
Can you summarize the main goal of these modules and the key difference compared to the basic centerline extraction?

**Comment by chir-set** [comment]
> Can you summarize the main goal of these modules and the key difference compared to the basic centerline extraction?

Fiducial centerline extraction allows to save time essentially, given a CT scan of *good* quality. An ROI is mandatory, this by itself guarantees processing speed. So we draw an ROI and place 2 or 3 fiducial points, we're done. We do no need a centerline to all branches inside the ROI bounds. We target here surgically relevant arteries (carotids, aorta, femorals) most commonly. Once again, given a CT scan of *good* quality. It should not be used for poor quality CT scans. This one below has been done in at most 1 min.
![FCE](https://user-images.githubusercontent.com/33979252/201475743-c000ad38-0dd8-4668-a2e9-b5dff1d7d6b9.png)


Curve centerline extraction targets difficult arterial segmentation : vertebrals, coronaries, distal arteries over short portions... It allows to save time also as it packs many steps in a single script (draw tube, split volume, segmentation and of course centerline extraction). It should not be used in a hurry like the previous one, but it allows work to be done. Of course, like any tool, it will not solve all tasks.
![CCE](https://user-images.githubusercontent.com/33979252/201475774-b4aeeb0d-4ed9-4d5d-80b4-9b918f62ea5d.png)

As for key difference with 'Extract centerline' module, there are none, and improving 'Extract centerline' was not a goal; these modules simply remote control 'Extract centerline'.

If you deem these modules do not fit in SlicerVMTK for whatever reasons and wish to remove them, it's OK for me. I'll make them generally available otherwise, that's the least problem.

Your decision will influence renaming and updating these modules or not.

**Comment by lassoan** [comment]
The modules look very useful and a good fit in the VMTK extension. We just need to improve how users can find the most relevant modules for their work.

Since the end goal of these higher-level modules is not centerline extraction, that is just one tool utilized during the workflow, the "centerline" and "extraction" words should be de-emphasized or removed from their names. These high-level modules could have "vessel analysis" in their names instead. 

The name should reflect the difference between the quick&easy and the one for the difficult cases. 

Maybe the first module could be named something like "Quick vessel analysis" and the other could be "Guided vessel analysis"?

Maybe they could be merged into one "Vessel analysis" module and the user could choose between the two methods in the module? This would make sense especially if there was significant code and workflow duplication between the two modules (but I'm not sure if this is the case).

Thanks again for all your contributions!

**Comment by chir-set** [comment]
> the "centerline" and "extraction" words should be de-emphasized or removed from their names

It's reasonable, the more so that centerline extraction is, not formally optional because it's a goal, but secondary to an acceptable segmentation.

The word 'vessel' itself may introduce confusion as it may suggest it's a first hand way of segmenting veins while it would rarely be successful there. Other options would be : 

{Quick, Guided} {tube, artery, arterial} {analysis, segmentation}


As for merging both, I don't think it's a good idea. They do not primarily target the same kind of arteries, nor do the internals overlap that much.

**Comment by chir-set** [comment]
The modules have been renamed : 

Fiducial centerline extraction : Quick artery segmentation
Curve centerline extraction : Guided artery segmentation

**Comment by lassoan** [comment]
Thank your, these new names sound good, they reflect very well their intended use.
